src/index.ts

```diff
@@ -1,26 +1,45 @@
 import * as core from '@actions/core'
 import * as github from '@actions/github'
-import { parseReferencedIssues, uniq } from './issue-parser'
+import {
+  createReferenceRegExp,
+  parseReferencedIssues,
+  uniq,
+} from './issue-parser'
+
+function getInputAsArray(name: string, options?: core.InputOptions): string[] {
```

Comment:
maybe, it could be a good idea to name what kind of input you want to get as an array.. however, this may not be true since I'm not sure what kind of input you're dealing with here.

Reply:
Input is input from workflow.yml